spec/index.html

```diff
@@ -1174,9 +1174,11 @@ <h2>Turtle Grammar</h2>
 
   <p>A <a>Turtle document</a> is an <a data-cite="RDF12-CONCEPTS#dfn-rdf-string">RDF string</a>
     encoded in UTF-8 [[!RFC3629]].
-  Only <a data-cite="I18N-GLOSSARY#dfn-code-point">Unicode code points</a>
-  in the range <code class="codepoint">U+0000</code> to <code class="codepoint">U+10FFFF</code>
-  inclusive are allowed.
+    Only <a data-cite="I18N-GLOSSARY#dfn-scalar-value" class="lint-ignore">Unicode scalar values</a>,
+    in the range <code class="codepoint">U+0000</code> to <code class="codepoint">U+D7FF</code> 
+    and  <code class="codepoint">U+E000</code> to <code class="codepoint">U+10FFFF</code> inclusive,
+    are allowed.
```

Comment:
```suggestion
    Only <a data-cite="I18N-GLOSSARY#dfn-scalar-value" class="lint-ignore">Unicode scalar values</a>
    in the ranges <code class="codepoint">U+0000</code> to <code class="codepoint">U+D7FF</code>
    and <code class="codepoint">U+E000</code> to <code class="codepoint">U+10FFFF</code>, inclusive,
    are allowed.
```

Reply:
"ranges" fixed.

Note:  Unicode does not say "inclusive" after every range mention so it could be dropped. A range is "a set of numbers that are valid". Intervals are open/closed.